src/pybind/mgr/dashboard/frontend/src/app/shared/components/pwd-expiry-notification/pwd-expiry-notification.component.html

```diff
@@ -0,0 +1,10 @@
+<alert class="no-margin-bottom" type="{{ alertType }}" *ngIf="expiryDays <= levels[warningLevel]" [dismissible]="true" i18n>
```

Comment:
```suggestion
<alert class="mb-0" type="{{ alertType }}" *ngIf="expiryDays <= levels[warningLevel]" [dismissible]="true" i18n>
```
Bootstrap4 provides some classes to set the margin and padding of elements.

Reply:
I changed it as you suggested but the result is not so nice I think. I'm not really sure what I'm doing wrong.

![Screenshot_2019-10-21_12-01-18](https://user-images.githubusercontent.com/8761082/67196302-d4425000-f3fa-11e9-8daa-967706f6578f.png)

Reply:
My bad, I tough the style was being applied to the element itself, but its on one of its children.
You can discard my suggestion.